spec/scenarios/credit_notes/credit_note_spec.rb

```diff
@@ -816,17 +825,17 @@
 
         estimate = json[:estimated_credit_note]
         expect(estimate).to include(
-          taxes_amount_cents: 1319,
+          taxes_amount_cents: 13_19,
           precise_taxes_amount_cents: "1319.25547",
-          sub_total_excluding_taxes_amount_cents: 6596,
-          max_creditable_amount_cents: 7916,
-          coupons_adjustment_amount_cents: 237,
+          sub_total_excluding_taxes_amount_cents: 65_96,
+          max_creditable_amount_cents: 79_15,
```

Comment:
I am trying to understand why these values changed 
aaaaah
aaah!!!!
Because in the previous approach, that we have at `credit_notes/create_service#adjust_credit_note_tax_rounding`, we were always adjusting the taxes, so it was not matematically correct, while now we're also adjusting total!
so definitely now we're using the `AdjustAmountsWithRoundingService`, so we can remove that method 🙌

Reply:
Actually, I still need to rely on the `precise_taxes_amount_cents` with the rounding adjustment.
When I get rid of it I end-up having a lot of validation issues when trying to create the credit notes